samples/bluetooth/hci_usb/src/main.c

```diff
@@ -451,6 +457,10 @@ static void btusb_bulk_out(u8_t ep, enum usb_dc_ep_cb_status_code ep_status)
 			net_buf_add_mem(buf, tmp, remains);
 		}
 	}
+#else
+	mem = (u8_t *)net_buf_add(buf, len);
+	usb_read(ep, mem, len, NULL);
```

Comment:
you do not need to cast to (u8_t *) here. You can also use 
`usb_read(ep, net_buf_add(buf, len), len, NULL);`

Reply:
Okay, that's much simpler. I'll rework and update the PR.